eventorganizer.api/src/main/java/com/event/organizer/api/model/Event.java

```diff
@@ -47,4 +49,5 @@ public class Event {
     @OneToMany(cascade=CascadeType.ALL)
     @JoinColumn(name="event_id")
     private List<Comment> comments;
+    private List<AppUser> interestedUsers;
```

Comment:
This should be @OneToMany, as we have mapping of One Event towards multiple Appusers